### PR TITLE
Create blog contributions guidelines

### PR DESCRIPTION
**Pull request by geriom**
# Changes

Create a new entry for blog posts in the contribution guidelines section.

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [x] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/website/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

File: content/en/docs/Contribute/blog-guidelines.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+1.	Below the front matter, write to your heart's content. Every time you save,
+    the server automatically builds the page and shows the changes.
+
+If you need some inspiration, you can find examples on the [website
+repository][blogs-folder].
+
+## Open a PR for review
+
+1.  After you complete your blog post, open a Pull Request and wait for it to
+    be reviewd.
```
Comment (AlanGreene) [inline content/en/docs/Contribute/blog-guidelines.md:84]:
```suggestion
    be reviewed.
```

File: content/en/docs/Contribute/blog-guidelines.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+
+1.  Your blog post is now public and we are all exited to read it.
```
Comment (AlanGreene) [inline content/en/docs/Contribute/blog-guidelines.md:87]:
```suggestion
1.  Your blog post is now public and we are all excited to read it.
```

File: content/en/docs/Contribute/blog-guidelines.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+    ```
+
+    Keep the server running while you write your post. Press *Ctrl + C* to stop
+    it.
+
+1.  Open your browser and go to `http://localhost:8888`.
+
+## Draft your blog post
+
+1.  Create a folder under `content/en/blog/<year>/` with a descriptive name
```
Comment (AlanGreene) [inline content/en/docs/Contribute/blog-guidelines.md:56]: Any issues with whitespace or special characters that users need to be aware of?

**Review by AlanGreene — COMMENTED** [review]
It'll be great to get contributions to the blog, looking forward to it.

**Review by vdemeester — APPROVED** [review]

File: content/en/docs/Contribute/blog-guidelines.md (hunk 3 above)
Comment (geriom) [reply content/en/docs/Contribute/blog-guidelines.md:56]: Everything should work, but for consistency with the rest of the guidelines, I added the usual "Lower case, use dashes to separate words".

**Comment by geriom** [comment]
@AlanGreene, I updated the PR with the suggested changes. Thanks!

**Comment by afrittoli** [comment]
Thanks for this @geriom.
We discussed about the blog during the last community WG https://docs.google.com/document/d/1g2HQk_4ypYEWPX2WjOFa6vOMpwYoRNprph_P09TS1UM/edit#bookmark=id.v0l40gitqaey

I think it's good to have these guidelines here. We should also have content-oriented guidelines in the community repo that can link to this for technical guidance about how to actually contribute the blog post.

File: content/en/docs/Contribute/blog-guidelines.md (hunk 3 above)
Comment (AlanGreene) [reply content/en/docs/Contribute/blog-guidelines.md]: Sounds good, thanks.

**Comment by AlanGreene** [comment]
Thanks Geri
/lgtm

**Comment by geriom** [comment]
> We should also have content-oriented guidelines in the community repo that can link to this for technical guidance about how to actually contribute the blog post.

Good idea. Let me  check the meeting notes and draft some baseline guidelines and open a PR on the community repo. We can go from there to decide what kind of posts we want to publish.

**Pull request merged**